src/auth/auth.js

```diff
@@ -152,7 +154,8 @@ Auth.prototype.logout = async function() {
 
 	// clear user and stored token
 	this.setUser(null)
-	localStorage.removeItem(TokenName)
+	sessionStorage.removeItem(TokenName)
```

Comment:
this and the next line into a function
*Auth.prototype.clearToken()

Reply:
done